packages/data-stores/src/contextual-help/contextual-help.tsx

```diff
@@ -1,6 +1,9 @@
 import { __ } from '@wordpress/i18n';
+import debugFactory from 'debug';
```

Comment:
So convenient that the `debug` dependency was already there!

Reply:
Yep! Apparently currently unused on `trunk` 😅